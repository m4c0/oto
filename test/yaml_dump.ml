module Printer = Outo.Yaml (Poc.Printer)

let () = Printer.print_yaml Poc.game
