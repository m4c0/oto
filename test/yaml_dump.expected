name: Title
background: title
music: romance-1
actors:
script:
outcome: jump to Intro
---
name: Intro
background: restaurant
music: romance-1
actors:
  left:
    - lefty
  middle:
    - midly
  right:
    - righty
script:
  - [lefty] Hello! How are you doing?
  - [lefty] Do you want to know what's happening?
  - [righty] Yeah, you! Baby, baby!
  - [midly] Click one option to select who you want to date
  - midly poses imponent
  - [midly] I'm as hot as an idol
outcome:
  - "Lefty" jumps to Lefty Intro
  - "Midly" jumps to Midly Intro
  - "Righty" jumps to Righty Intro
---
name: Lefty Intro
actors:
script:
outcome: pause then move to The End
---
name: The End
background: end
music: game-over
actors:
script:
outcome: end game
---
name: Midly Intro
actors:
  left:
    - midly
script:
outcome: pause then move to The End
---
name: Righty Intro
actors:
script:
outcome: pause then move to The End
---
